Admin/AbstractAdmin.php

```diff
@@ -1701,7 +1701,7 @@ public function getSubject()
      */
     public function hasSubject()
     {
-        return $this->subject != null;
+        return $this->getSubject() != null;
```

Comment:
Casting to boolean instead might be better, but this was not introduced by @phansys